point.py

```diff
@@ -0,0 +1,29 @@
+class Point(object):
+    def __init__(self,x,y,mark={}):
+        self.x = x
+        self.y = y
+        self.mark = mark
+
+    #implement magic method to add points (x,y)'s
+    def __add__(self,other):
+        return Point(self.x + other.x,self.y + other.y)
+    def __radd__(self,other):
+        return Point(self.x + other, self.y + other)
+
+    def __eq__(self,other):
+        return self.x == other.x and self.y == other.y
+
+    def __neq__(self,other):
+        return self.x != other.x or self.y != other.y
+
+    def patched_coincident(self,point2):
```

Comment:
I would not call these `patched_`.  Does your user really need to know that you patched these in?  It would be akin to saying `ClassFactory_Foo` in a very OO design.